arduino/smartcar/smartcar.ino

```diff
@@ -437,8 +436,14 @@ void autoRightPark(){ // the car is supposed to park inside a parking spot to it
         frontTimer++;
         distanceTraveled = newDistanceTraveled;
     }
-    car.setAngle(0);
-    car.setSpeed(10);
+    turningAngle = 0;
+    car.setAngle(turningAngle);
+    car.setSpeed(PARKING_SPEED);
+    while (!isObsAtFront(20)){
```

Comment:
remove curly brakcets, and perhaps add a comment to further elaborate as to what this is doing

Reply:
Fixed with https://github.com/DIT113-V22/group-02/pull/111/commits/590e4f481dd88a18fc2cef463aa02616c63bbbbb